d2l-hm-constants-behavior.html

```diff
@@ -12,6 +12,8 @@
 	window.D2L.Hypermedia.HMConstantsBehavior = {
 		HypermediaRels: {
 			// default Brightspace domain rels
+			activityUsage: 'https://api.brightspace.com/rels/activity-usage',
```

Comment:
Hmm, this feels like we put this in the wrong place - I suspect we should have done this within the activities domain. I'll make a note to fix that in the API.